plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -963,6 +963,37 @@ public void testTableComments()
         dropTable("iceberg.tpch.test_table_comments");
     }
 
+    @Test
+    public void testCreateTableWithLocationAndSchemaNotExists()
+    {
+        File tempDir = getDistributedQueryRunner().getCoordinator().getBaseDataDir().toFile();
+        String tempDirPath = tempDir.toURI().toASCIIString() + randomTableSuffix();
+        String createTableWithLocationTemplate = "" +
+                "CREATE TABLE iceberg.any_schema.test_table_comments (\n" +
+                "   _x bigint\n" +
+                ")\n" +
+                "WITH (\n" +
+                format("   format = '%s',\n", format) +
```

Comment:
I would remove this line because the table format unrelates to what we want to verify in this test and it's specified in `createQueryRunner()`.